README.md

```diff
@@ -145,6 +145,46 @@ repos:
         args: ["--enable require-variable-braces,deprecate-which"]
 ```
 
+## tflint Caveats
+
+### Using the `--config` argument
+
+With the introduction of `--chdir` into tflint, the `--config` argument is now bound to whatever subdirectory you are
+running the check against.  For mono-repos this isn't ideal as you may have a central configuration file you'd like to
+use.  If this matches your use-case, you can specify the placeholder `__GIT_DIR__` value in the `--config` argument 
```

Comment:
Instead of deferring to a manual adjustment for __GIT_DIR__, it would probably be preferred to have the hooks/tflint.sh script populate this automatically. Would it be possible for you to make this adjustment so as to minimize the number of manual changes required?